### PR TITLE
Some minor bug fixes (Suzu)

### PR DESCRIPTION
**Pull request by BlueFunny19**
PR Introduction: https://github.com/Gakuto1112/SenkoSan/pull/15
This PR is the last PR to fix some problems with `Suzu` branching.

### CODE REVIEW

**Comment by Gakuto1112** [comment]
Thanks for your pull request! I will review this later. However, I'm busy for another project that has a deadline, so it may takes long time to deal with it. I will do it as soon as possible.

Some codes in Suzu are same as Senko's codes. I will resolve this pull request by merging Senko's codes if I can do it.

Thanks for your corporation!

File: scripts/general.lua [hunk 1]
```diff
@@ -79,4 +79,21 @@ events.TICK:register(function ()
...
+    end
+    if context == "PAPERDOLL" then
+        shouldRenderHead = true
+    end
+    if context == "MINECRAFT_GUI" then
+        shouldRenderHead = true
+    end
+    models.models.main.Avatar.Head:setVisible(shouldRenderHead)
+end)
+
```
Comment (Gakuto1112) [inline scripts/general.lua:95]: The same problem that I mentioned in #15 is occurred. Can you fix this?

**Review by Gakuto1112 — CHANGES_REQUESTED** [review]
With your changes, Figura no longer emits errors. However, the same problem that I mentioned in #15 is occurred. Can you fix this?

**Review by Gakuto1112 — CHANGES_REQUESTED** [review]
I mentioned about these changes [here](https://github.com/Gakuto1112/SenkoSan/pull/15#pullrequestreview-1918626595). Please check it.

**Comment by BlueFunny19** [comment]
See here: https://github.com/Gakuto1112/SenkoSan/pull/15#issuecomment-2441573578

**Pull request closed without merging**